include/qore/intern/qore_program_private.h

```diff
@@ -2110,6 +2121,17 @@ class qore_program_private : public qore_program_private_base {
       }
    }
 
+   DLLLOCAL void getStatementBreakpoints(const AbstractStatement* statement, QoreBreakpointList_t &bkptList) {
+      QoreAutoRWReadLocker al(&lck_breakpoint);
+      bkptList.clear();
+	  if (statement->breakpoints) {
+	     for (std::list<QoreBreakpoint*>::iterator it = statement->breakpoints->begin(); it != statement->breakpoints->end(); ++it) {
+		    bkptList.push_back(*it);
+			(*it)->ref();
```

Comment:
remove the tab char
fyi, you can use `auto` instead of `std::list<QoreBreakpoint*>::iterator`
or even `for *const auto & breakpoint : statement->breakpoints) {`